### ISSUE TITLE
Cura 3.2.0 and Sigma R17 updated R19 FW01-2.0.8

### ISSUE THREAD

**Issue by tilowil** [BCN3D/BCN3DSigma-Firmware#229]
Hello everybody,
In first, I am sorry for my poor english. I speak Google Traduction English.
Before explaining the problem to you, I give you all the technical information.
I have a BCN Sigma R17 updated to R19 with a firmware 01-2.0.8 updated through BCN3D Cura downloaded from the site bcn3d.com. The version of Cura is 3.2.0 which works in an MS Windows 10 environment.
I defined the materials in cura with their cost, their weight and the temperature of use and that of the bed.
Still in Cura, I defined a Sigma R19 as a printer. I haven’t changed any parameters anywhere except the filament temperatures.
Before this big update, I used my Sigma without problem with Cura 0.1.5 beta2 and firmware 01-1.0.2.
Now that the technical context is in place, the problem:
I would like to test before resuming my prints, because with the new E3D nozzles some parameters may have to be changed. I took 3DBenchy, I chose extruder 1, a yellow PLA filament (extruder: 190°C, bed: 50°C). I prepare the slicing. I record on the SD. I put the SD in the printer. I launch the print and I realize that the temperature of the bed is increasing, but not that of the left extruder (nor that of the right extruder). I go to the print settings (then temperatures) on the Sigma and I see the following temperatures:
– Left extruder: 0°C
– Bed: 60°C
– Right extruder: 0°C
I stop printing. I tell myself that the GCode did not have the information. I open it:
—————————————————————————————————————————————————
;FLAVOR:Marlin
;TIME:14160
;Filament used: 1.6062m
;Layer height: 0.1
;Extruders used: T0 0.4
;Materials used: T0 PLA
;BCN3D_FIXES
;Generated with Cura_SteamEngine 3.2.0
M141 S50
T0
M190 S60
M104 S190
M109 S190
M82 ;absolute extrusion mode
;Sigma ProGen 2.2.0 (Build 24HJ1318)
;BCN3D Fixes applied
; – Fix Acceleration/Jerk commands
; – Z Hop At Layer Change

G21 ;metric values
G90 ;absolute positioning
M204 S600 ;set default acceleration
M205 X12.5 Y12.5 ;set default jerk
M107 ;start with the fan off
G28 X0 Y0 ;move X/Y to min endstops
G28 Z0 ;move Z to min endstops
G1 Z5 F200 ;safety Z axis movement
T1 ;switch to the right extruder
G92 E0 ;zero the extruded length
G1 E10 F35
G92 E0
G1 E-6.5 F1000
G92 E0
T0 ;switch to the left extruder
G92 E0 ;zero the extruded length
G1 E10 F35
G92 E0
G4 S2 ;stabilize hotendG1 E-0.5 F1000
G92 E0

G4 P1
G4 P2
G4 P3

;LAYER_COUNT:480
;LAYER:0
M107
M204 S580
M205 X9.38 Y9.38
G0 F2700 X104.926 Y164.551 Z0.2
M204 S435
;TYPE:SKIRT
G1 F900 X104.282 Y164.538 E0.00969
G1 X80.146 Y163.07 E0.37357
—————————————————————————————————————————————————
and unless I am mistaken the information is well transmitted, well almost, because I request a bed at 50°C and there is a wait for a bed (M190) at 60°C, and I do not understand the M141 S50, n shouldn’t it have been M140 S50?
Maybe the semicolon at the end of the line is missing for M190, M141, M104, M109 and T0 in the lines generated by the Cura_SteamEngine 3.2.0.
In short, I can print but by adjusting the temperatures manually, it’s a bit stupid.
How to fix this problem ?

Thank you for your help.
